modules/cli-options/src/main/scala/scala/cli/util/ArgParsers.scala

```diff
@@ -0,0 +1,27 @@
+package scala.cli.util
+
+import caseapp.core.argparser.ArgParser
+import caseapp.core.argparser.SimpleArgParser
+
+import scala.build.options.publish.MaybeConfigPasswordOption
+import scala.cli.signing.shared.PasswordOption
+
+abstract class LowPriorityArgParsers {
```

Comment:
thinking if this file could use some scaladocs with extra context (i.e. on how it differs from `caseapp`'s arg parsers)